<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageReplacement</class>
 <widget class="QMainWindow" name="PageReplacement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PB_run">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>469</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>模拟</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zhenshu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>469</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>内存块数：</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackWid">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>52</y>
      <width>677</width>
      <height>382</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QTableView" name="taVi_opt">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTableView" name="taVi_fifo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QTableView" name="taVi_lru">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:&quot;transparent&quot;;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="PB_qufilecreate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>访问串创建</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_kuaishu">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>469</y>
      <width>169</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_qufile">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>441</y>
      <width>169</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>440</y>
      <width>256</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:&quot;transparent&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_allclear">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>469</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_opt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>498</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OPT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_fifo">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>498</y>
      <width>169</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FIFO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_openfile">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_lru">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>498</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LRU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_onerun">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>498</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>单步模拟</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_qufile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>访问串文件： </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
